src/fold.rs

```diff
@@ -500,39 +502,37 @@ fn shift_right(
 fn shift_left(
     left: Expr,
     right: Expr,
-    ctype: &Type,
+    _ctype: &Type,
```

Comment:
If you're not going to use the type, please remove it from the function signature.